gatsby-config.js

```diff
@@ -58,55 +58,6 @@ module.exports = {
                     }
                 ]
             },
-            {
-                title: "Version 4 Mobile SDKs end-of-support FAQ",
-                path: "/documentation/v4-end-of-life-faq"
-            },
-            {
-                title: "Version 4 API changelog",
-                path: "/documentation/api-changelog"
-            },
```

Comment:
I am thinking we can keep these two here, @swarna04 thoughts on this?

Reply:
Until we get rid of them, let's have it in both places. Sounds good?